api/example.com/resource/gpu/v1alpha1/api.go

```diff
@@ -40,6 +42,6 @@ func DefaultDeviceClassParametersSpec() *DeviceClassParametersSpec {
 
 func DefaultGpuClaimParametersSpec() *GpuClaimParametersSpec {
 	return &GpuClaimParametersSpec{
-		Count: 1,
+		Count: ptr.To(1),
```

Comment:
Why?

There's probably a reason, I just don't see it mentioned in the commit message and don't know it.

Reply:
This will be important once I add the `selection` APIs via vendor-specific CRDs. It is an artifact of the fact that I pulled this structured parameters code over from the implementation I did in our GPU driver and this is how the `Count` field exists over there. I could hold off on this change until I add the selector code if that's prefereble.

Reply:
Whatever works for you best. I was just wondering.